livebooks/elixir_ASCII_heart.livemd

```diff
@@ -0,0 +1,72 @@
+# Elixir goes ASCII Art ¯\_(ツ)_/¯
+
+## We💜Elixir, right?
+
+The aim is to create a heart-shaped pattern in your terminal that is made up of characters from the given string _Elixir_.
+
+The following code achieves that by using a combination of mathematical equations and string manipulation techniques.
+
+You'll find a step-by-step breakdown after the code. Let's go!
+
+```elixir
+defmodule Love do
+  @love "Elixir "
+
+  def character_at(@love, x, y) do
+    index = rem(x - y, String.length(@love))
+    char = String.slice(@love, index..index)
+
+    if (:math.pow(x * 0.05, 2) + :math.pow(y * 0.1, 2) - 1) ** 3 -
```

Comment:
For better readability i would put that in its own function then you can write if(something?) do.....

Reply:
or even case something? do true -> ... _other -> ....